openedx_external_enrollments/tests/external_enrollments/tests_pathstream_external_enrollments.py

```diff
@@ -0,0 +1,97 @@
+
+
+class PathstreamExternalEnrollmentTest(TestCase):
+    """Test class for PathstreamExternalEnrollment."""
+
+    def setUp(self):
+        """Set test database."""
+        self.base = PathstreamExternalEnrollment()
+
+    def test_str(self):
+        """
+        PathstreamExternalEnrollment overrides the __str__ method,
+        this test that the method __str__ returns the right value.
+        """
+        self.assertEqual(
+            'pathstream',
+            self.base.__str__(),
+        )
+
+    @patch('openedx_external_enrollments.external_enrollments.pathstream_external_enrollment.datetime')
+    def test_get_enrollment_data(self, datetime_mock):
+        """Testing _get_enrollment_data method."""
+        course_settings = None
+        data = {
+            'user_email': 'javier12@gmail',
+            'is_active': True,
+            'course_id': 'courseTest-v1:XX+SC1001+202X_TX'
+        }
+        datetime_mock.utcnow.return_value.strftime.return_value = '2021-06-28 16:40:31.456900'
+
+        expected_data = 'courseTest-v1:XX+SC1001+202X_TX,javier12@gmail,2021-06-28 16:40:31.456900,true\n'
+
+        result = self.base._get_enrollment_data(data, course_settings)  # pylint: disable=protected-access
+
+        self.assertEqual(
+            expected_data,
+            result,
+        )
+
+    @patch(
+        'openedx_external_enrollments.external_enrollments.'
+        'pathstream_external_enrollment.PathstreamExternalEnrollment._get_enrollment_data'
+    )
+    def test_post_enrollment(self, get_enrollment_data_mock):
+        """Test _post_enrollment method."""
+        course_settings = None
+        course_overview = CourseOverview()
+        course_overview.save()
+        data = {
+            'user_email': 'javier12@gmail',
+            'is_active': True,
+            'course_id': course_overview.id  # pylint: disable=no-member
+            # Normally course_id should be a CouseLocator
+        }
+        enrollment_data = 'courseTest-v1:XX+SC1001+202X_TX,javier12@gmail,2021-06-28 16:40:31.456900,true\n'
+        get_enrollment_data_mock.return_value = enrollment_data
+        meta_expected = {
+            'enrollment_data_formated': enrollment_data,
+            'is_uploaded': False,
+        }
+
+        log1 = 'calling enrollment for [{}] with data: {}'.format(self.base.__str__(), data)
+        log2 = 'calling enrollment for [{}] with course settings: {}'.format(self.base.__str__(), course_settings)
+
+        with LogCapture(level=logging.INFO) as log_capture:
+            self.base._post_enrollment(data, course_settings)  # pylint: disable=protected-access
+
+            enrollment = ExternalEnrollment.objects.get(  # pylint: disable=no-member
+                email=data['user_email'],
+                course_shell=data['course_id']
+            )
+            log3 = 'Saving External enrollment object for [{}]' \
+                   ' -- ExternalEnrollment.id = {}'.format(self.base.__str__(), enrollment.id)
```

Comment:
```suggestion
            log3 = 'Saving External enrollment object for [{}] -- ExternalEnrollment.id = {}'.format(
                self.base.__str__(), 
                enrollment.id,
            )
```
Also add a blank line to separate action from assignment statements.